### ISSUE TITLE
[IEEE Access(2020)] 実世界重みクロスエントロピー損失関数 誤表示のコストをモデル化する

### ISSUE THREAD

**Issue by yoshidaKN** [yoshidaKN/Paper-CV-NLP#1]
# 情報
- **タイトル**  
The Real-World-Weight Cross-Entropy Loss Function: Modeling the Costs of Mislabeling
- **日本語訳タイトル**  
実世界重みクロスエントロピー損失関数： 誤表示のコストをモデル化する
- **採択・年数**  
IEEE(2020)
- **URL**  
https://arxiv.org/abs/2001.00570
- ** ラベル不均衡問題や、誤判定時の実世界で生じるコストをそのまま再現して運用する手法 **
